tests/console/commands/test_add.py

```diff
@@ -347,6 +347,18 @@ def test_add_constraint_dependencies(tester: CommandTester) -> None:
     assert tester.command.installer.executor.installations_count == 2
 
 
+def test_add_with_markers(app: PoetryTestApplication, tester: CommandTester) -> None:
+    marker = "python_version <= '3.4' or sys_platform == 'win32'"
+    tester.execute(f"""cachy --markers "{marker}" """)
```

Comment:
Single quotes are not sufficient here, because the markers itself must be quoted, e.g. `poetry add cachy --markers "python_version <= '3.4' or sys_platform == 'win32'"`

Reply:
Ok, I checked that and with single quotes the marker string gets escaped, so let's skip that I guess.